azurerm/internal/services/logic/tests/logic_app_workflow_resource_test.go

```diff
@@ -198,3 +238,71 @@ resource "azurerm_logic_app_workflow" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
+
+func testAccAzureRMLogicAppWorkflow_integrationAccount(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%[1]d"
```

Comment:
We should add the package here too so it's easier to know where potential issues are coming from when looking at the Azure Portal 
```suggestion
  name     = "acctestRG-logic-%[1]d"
```